examples/old-examples/README.md

```diff
@@ -38,86 +38,67 @@ pip install PySDL2
+  - [particle_system_2](GLWindow/particle_system_2.py)
+  - [pillow_font](GLWindow/pillow_font.py)
+  - [using_texture3d](GLWindow/using_texture3d.py)
+  - [perspective_projection](GLWindow/perspective_projection.py)
+  - [window_coordinates](GLWindow/window_coordinates.py)
+
+- [Kivy](Kivy)
+  - [01_hello_world](Kivy/01_hello_world.py)
+  - [perspective_projection](Kivy/perspective_projection.py)
+  - [window_coordinates](Kivy/window_coordinates.py)
+
+- [pygame](pygame)
+  - [01_hello_world](pygame/01_hello_world.py)
+  - [perspective_projection](pygame/perspective_projection.py)
+  - [window_coordinates](pygame/window_coordinates.py)
+
+- [pyglet](pyglet)
+  - [01_hello_world](pyglet/01_hello_world.py)
+  - [02_uniforms_and_attributes](pyglet/02_uniforms_and_attributes.py)
+  - [03_alpha_blending](pyglet/03_alpha_blending.py)
+  - [04_textures](pyglet/04_textures.py)
+  - [perspective_projection](pyglet/perspective_projection.py)
+  - [window_coordinates](pyglet/window_coordinates.py)
+
+- [PyQt5](PyQt5)
+  - [01_hello_world](PyQt5/01_hello_world.py)
+  - [02_uniforms_and_attributes](PyQt5/02_uniforms_and_attributes.py)
+  - [03_alpha_blending](PyQt5/03_alpha_blending.py)
+  - [04_textures](PyQt5/04_textures.py)
+  - [05_perspective_projection](PyQt5/05_perspective_projection.py)
+  - [perspective_projection](PyQt5/perspective_projection.py)
+  - [window_coordinates](PyQt5/window_coordinates.py)
+
+- [PyQt5](PySDL2)
   - soon
 
-- [tkinter](https://github.com/cprogrammer1994/ModernGL/tree/master/examples/tkinter)
-  - [01_hello_world](https://github.com/cprogrammer1994/ModernGL/tree/master/examples/tkinter/01_hello_world.py)
-  - [colors](https://github.com/cprogrammer1994/ModernGL/tree/master/examples/tkinter/colors.py)
+- [tkinter](tkinter)
+  - [01_hello_world](tkinter/01_hello_world.py)
+  - [colors](tkinter/colors.py)
 
-- [standalone](https://github.com/cprogrammer1994/ModernGL/tree/master/examples/standalone)
-  - [01_hello_world](https://github.com/cprogrammer1994/ModernGL/tree/master/examples/standalone/01_hello_world.py)
-  - [render_to_pillow_image](https://github.com/cprogrammer1994/ModernGL/tree/master/examples/standalone/render_to_pillow_image.py)
-  - [render_to_pillow_image_multisample](https://github.com/cprogrammer1994/ModernGL/tree/master/examples/standalone/render_to_pillow_image_multisample.py)
-  - [perspective_projection](https://github.com/cprogrammer1994/ModernGL/tree/master/examples/standalone/perspective_projection.py)
-  - [window_coordinates](https://github.com/cprogrammer1994/ModernGL/tree/master/examples/standalone/window_coordinates.py)
+- [standalone](standalone)
+  - [01_hello_world](standalone/01_hello_world.py)
+  - [render_to_pillow_image](standalone/render_to_pillow_image.py)
+  - [render_to_pillow_image_multisample](standalone/render_to_pillow_image_multisample.py)
+  - [perspective_projection](standalone/perspective_projection.py)
+  - [window_coordinates](standalone/window_coordinates.py)
 
```

Comment:
The following images were all broken links. I can't find exactly corresponding images in the repo any more, so I deleted this section.